### PR TITLE
Rename collaborator/s to person/people

### PR DESCRIPTION
**Pull request by PVince81**
## Description
Rename "collaborator" to "person".
Rename "collaborators" to "people".
Rename "Add Collaborator" to "share".

## Related Issue
Part 1 and part 4a of https://github.com/owncloud/product/issues/231

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by PVince81** [comment]
changelog added.

let's hope tests passed as I had to change some labels there

File: apps/files/src/components/Collaborators/Collaborator.vue [hunk 1]
```diff
@@ -69,7 +69,7 @@
         </oc-button>
         <oc-spinner
           v-else-if="$_loadingSpinnerVisible"
-          :aria-label="$gettext('Removing collaborator…')"
+          :aria-label="$gettext('Removing person…')"
```
Comment (LukasHirt) [inline apps/files/src/components/Collaborators/Collaborator.vue:72]: When adding new people, the spinner has label without those three dots. Can we remove them here as well for consistency?

File: apps/files/src/components/Collaborators/NewCollaborator.vue [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="files-collaborators-collaborator-add-dialog">
-    <label for="oc-sharing-autocomplete"><translate>New Collaborators:</translate></label>
+    <label for="oc-sharing-autocomplete"><translate>New People:</translate></label>
```
Comment (LukasHirt) [inline apps/files/src/components/Collaborators/NewCollaborator.vue:3]: "New People", looks a bit strange. Would maybe "Add People" fit here better?

**Comment by PVince81** [comment]
this will take a bit longer... seems some sidebar tab selectors are based on the text given in Gherkin scenarios

**Review by LukasHirt — APPROVED** [review]
Just two small comments. More questions then requested changes :wink:

**Comment by phil-davis** [comment]
> this will take a bit longer... seems some sidebar tab selectors are based on the text given in Gherkin scenarios

Steps like this seem to get confused:
```
    When the user switches to "collaborators" tab in details panel using the webUI
    Then the "collaborators" details panel should be visible
```

But I don't immediately see which thing changed that breaks that test code.

**Comment by PVince81** [comment]
@phil-davis already on it.

the first row uses a selector that hard-codes the panel name into the selector.
the second row uses an if statement...

I have a local fix, testing now... then will push.

File: apps/files/src/components/Collaborators/Collaborator.vue (hunk 1 above)
Comment (PVince81) [reply apps/files/src/components/Collaborators/Collaborator.vue]: done

File: apps/files/src/components/Collaborators/NewCollaborator.vue (hunk 2 above)
Comment (PVince81) [reply apps/files/src/components/Collaborators/NewCollaborator.vue]: done

**Comment by PVince81** [comment]
I've fixed the tests, they passed locally. Let's see.

**Pull request merged**